modules/@apostrophecms/attachment/index.js

```diff
@@ -712,11 +709,26 @@ module.exports = {
             if (options.annotate) {
               // Add URLs
               value._urls = {};
+              if (value._crop) {
+                value._urls.uncropped = {};
```

Comment:
Feels weird to have the `uncropped` property inside `_urls`, which results in adding deep complex structure.

I did that because I thought that was a "dirty" workaround at first, but since we're going to keep that, maybe we should have a `_uncroppedUrls` ou `_urlsUncropped` property next to `_urls` to keep it cleaner:

```
{
  _urls: {
    // ...
  },
  _urlsUncropped: {
    // ...
  },
```

wdyt?

Reply:
I'm even not sure we need all the uncropped version here, but feeled logical.
I would say both are ok for me, if you feel more logical to have a specific property for uncropped ones it's good for me.